pkg/storaged/client.js

```diff
@@ -475,21 +473,21 @@
             function (success) {
                 // hack here
                 client.features.legacy_vdo = success;
-                return cockpit.resolve();
+                return Promise.resolve();
             },
             function () {
-                return cockpit.resolve();
+                return Promise.resolve();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)